service/src/test/java/bio/terra/pipelines/common/utils/PipelineInputTypesEnumTest.java

```diff
@@ -122,17 +160,44 @@ private static Stream<Arguments> castValidations() {
 
   @ParameterizedTest
   @MethodSource("castValidations")
-  void castInputValues(
+  <T> void castInputValues(
+      PipelineInputTypesEnum inputType, Object inputValue, T expectedCastValue) {
+    if (expectedCastValue != null) {
+      if (expectedCastValue instanceof List expectedListCastValue) {
+        List<String> listCastValue =
+            inputType.cast("fieldName", inputValue, new TypeReference<>() {});
+        assertTrue(expectedListCastValue.containsAll(listCastValue));
```

Comment:
can we do `assertEquals(expectedListCastValue, listCastValue)` here? I'm not sure `containsAll` is the right test on its own here.

Reply:
huh, yes, previously the way i was doing this the assertEquals on lists didn't work but now it does. thanks!